sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MeterProviderConfiguration.java

```diff
@@ -40,14 +43,26 @@ static void configureMeterProvider(
         break;
     }
 
-    String exporterName = config.getString("otel.metrics.exporter");
-    if (exporterName == null) {
-      exporterName = "otlp";
+    Set<String> exporterNames = DefaultConfigProperties.getSet(config, "otel.metrics.exporter");
+    if (exporterNames.contains("none")) {
+      if (exporterNames.size() > 1) {
+        throw new ConfigurationException(
+            "otel.metrics.exporter contains none along with other exporters");
+      }
+      return;
     }
-    if (!exporterName.equals("none")) {
-      MetricExporterConfiguration.configureExporter(
-          exporterName, config, serviceClassLoader, meterProviderBuilder, metricExporterCustomizer);
+
+    if (exporterNames.isEmpty()) {
+      exporterNames = Collections.singleton("otlp");
     }
+    exporterNames.forEach(
+        exporterName ->
+            MetricExporterConfiguration.configureExporter(
+                exporterName,
+                config,
+                serviceClassLoader,
+                meterProviderBuilder,
+                metricExporterCustomizer));
```

Comment:
Let's break this out into its own function that returns `List<MetricReader>`:
```
  static List<MetricReader> configureMetricReaders(ConfigProperties config, ClassLoader serviceClassLoader,
      BiFunction<? super MetricExporter, ConfigProperties, ? extends MetricExporter>
          metricExporterCustomizer) {
    Set<String> exporterNames = DefaultConfigProperties.getSet(config, "otel.metrics.exporter");
    if (exporterNames.contains("none")) {
      if (exporterNames.size() > 1) {
        throw new ConfigurationException(
            "otel.metrics.exporter contains none along with other exporters");
      }
      return Collections.emptyList();
    }

    if (exporterNames.isEmpty()) {
      exporterNames = Collections.singleton("otlp");
    }

    return exporterNames.stream().map(exporterName ->
        MetricExporterConfiguration.configureExporter(exporterName, config, serviceClassLoader, metricExporterCustomizer))
            .collect(toList());
  }
```
And adjust `MetricExporterConfiguration.configureExporter` to return a `MetricReader`.

This will make it simpler to test.

Reply:
Make sense. I'll modify it.